### PR TITLE
MailNotifier: Provide the possibility to send HTML emails

### PR DESCRIPTION
**Pull request by MarcelBochtler**
Enable the mail notifier to be able to send HTML mails.

### CODE REVIEW

File: notifier/src/main/kotlin/modules/MailNotifier.kt [hunk 1]
```diff
@@ -19,34 +19,42 @@
...
 
     init {
-        client = SimpleEmail()
-
-        client.setHostName(config.hostName)
+        client.hostName = config.hostName
         client.setSmtpPort(config.port)
         client.setAuthenticator(DefaultAuthenticator(config.username, config.password))
-        client.setSSLOnConnect(config.useSsl)
+        client.isSSLOnConnect = config.useSsl
```
Comment (sschuberth) [inline notifier/src/main/kotlin/modules/MailNotifier.kt:39]: Nit: Unrelated change?

Comment (MarcelBochtler) [reply notifier/src/main/kotlin/modules/MailNotifier.kt:39]: This was not possible with the `Email` interface.
it is only possible with `HtmlEmail`. Should I move it to a separate commit?

Comment (sschuberth) [reply notifier/src/main/kotlin/modules/MailNotifier.kt:39]: Interesting that this is only possible with `HtmlEmail`. If you want, you could move this to a succeeding commit with an explanation that this is now possible but wasn't before. But maybe that's overdoing it; I'd also be fine to keep it as-is.

**Review by sschuberth — APPROVED** [review]
